config.toml

```diff
@@ -40,34 +41,77 @@ pathToRepo = ""
 # Ignored when `mode != "docker"`
 # Override by setting `dockerImage` env variable
 ##
-dockerImage = ''
+dockerImage = 'mojaloop/central-ledger:latest'
 
 ##
 # The list of repos to scan
-
 # Lines starting with a # will be ignored
 # Ignored when `mode != "standalone"`
 ##
-repos = [ ]
-
+repos = [
```

Comment:
I guess we don't need central-end-user-registry, central-hub..

Reply:
Cool cool.